string audio_topic_name
string image_topic_name
int32 queue_size
string file_name # This will be used as key
string file_format
string audio_format
string audio_sample_format
string audio_channels
string audio_depth
string audio_sample_rate
string video_encoding
string video_height
string video_width
string video_framerate
